### DISCUSSION TITLE
delete

### DISCUSSION THREAD

**Discussion by kamajoy** [apache/apisix#10440] [General]
已解决

**Comment by soutong** [comment]
什么问题

**Comment by juzhiyuan** [comment]
@kamajoy Can you please confirm is it a bug or configuration setting?

**Comment by flylan** [comment]
A meaningless question

**Comment by juzhiyuan** [comment]
Hi, if you want to know the original question, click:

<img width="951" alt="image" src="https://github.com/apache/apisix/assets/2106987/507b0c5e-40eb-498b-af65-4c44a67e33f2">
